package.json

```diff
@@ -29,7 +29,7 @@
       "@folio/stripes-template-editor"
     ],
     "okapiInterfaces": {
-      "circulation": "9.0",
+      "circulation": "9.0 10.0",
```

Comment:
Hello @zburke thank you
I may be wrong, but override-check-out-by-barcode was only used in ui-checkout and not used in other folio ui modules. override-check-out-by-barcode was removed from ui-checkout in Q12021 after "migration" to other endpoint.
Could you please clarify how i understood that we can migrate from version 9 to version 10 without support previous version ("circulation": "10.0",)?